src/app/shared/model/audio-version.model.ts

```diff
@@ -11,7 +11,7 @@ export class AudioVersionModel extends BaseModel {
   public id: number;
   public uploads: string = '';
   public label: string;
-  public explicit: string = '';
+  public explicit: string = null;
```

Comment:
How can we determine that a value has not been set and to do the override if we default it to one of the available values?